client/app/templates/show-project.hbs

```diff
@@ -117,7 +117,7 @@
                             content='City Planning Commision Report'
                             tagName='span'
                           }}
-                            <a href={{build-url "cpcReport" action.dcpUlurpnumber}} target="_blank">
+                            <a href={{if (and (not-eq action.statuscode ACTION_STATUSCODE_ACTIVE) action.dcpSpabsoluteurl) (build-url "cpcReport" action.dcpUlurpnumber action.dcpSpabsoluteurl)}} target="_blank">
```

Comment:
Hmm, this might be better if the if statement is outside the `<a>`. It would be a bit more readable imo, and also here it's not clear what happens if the condition is false. Would the link just go to nowhere?

Reply:
that's a good point! I'll make those changes